conf/postgres-operator/pgo.yaml

```diff
@@ -17,6 +17,8 @@ Cluster:
   BackrestS3Bucket:
   BackrestS3Endpoint:
   BackrestS3Region:
+  BackrestS3URIStyle: "host"
+  BackrestS3VerifyTLS: "y"
```

Comment:
Per earlier comment, it would be good to keep this as a boolean, (e.g. `like DisableAutofail` below) and then map it to `y`/`n` within the places that call pgBackRest

Reply:
Sure thing. All the configs are consistent in this regard now, BackrestS3VerifyTLS will take true/false no matter where it is configured. Behind the scenes, this will then be parsed, as needed, for the boolean value before the final 'n'/'y' value is assigned to the required variable.